bin/git-forgit

```diff
@@ -716,9 +719,12 @@ _forgit_branch_delete() {
         $FORGIT_BRANCH_DELETE_FZF_OPTS
     "
 
-    cmd="git branch --color=always | LC_ALL=C sort -k1.1,1.1 -rs"
-    for branch in $(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf | awk '{print $1}'); do
-        _forgit_git_branch_delete "$branch"
+    for branch in $(git branch --color=always |
+        LC_ALL=C sort -k1.1,1.1 -rs |
+        FZF_DEFAULT_OPTS="$opts" fzf |
+        awk '{print $1}')
+    do
+        _forgit_branch_delete "$branch"
```

Comment:
I think this should stay `_forgit_git_branch_delete`, shouldn't it?

Reply:
Yes, good catch!